Scoop/Controller/ProfileViewController.swift

```diff
@@ -57,7 +57,8 @@ class ProfileViewController: UIViewController, ProfileViewDelegate {
         getUserPreferences()
         setupHeaderImage()
         setupContainerView()
-        setupEditButton()
+        // Commented out because editing functionality is not yet implemented for MVP.
```

Comment:
When things are commented out like this, I recommend using a `MARK` or `TODO` or `FIXME` comment so that it's easy to pinpoint with Xcode, especially if another person works on it.